lib/Autocomplete.js

```diff
@@ -55,7 +56,7 @@ let Autocomplete = React.createClass({
   },
 
   componentWillMount () {
-    this.id = lodash.uniqueId('autocomplete-'); 
+    this.id = this.props.id || lodash.uniqueId('autocomplete-'); 
```

Comment:
https://github.com/reactjs/react-autocomplete/pull/89 appears to eliminate the label and `labelText` prop, which eliminates the need for a random id.  If that gets merged as it stands now, then this change is probably not necessary anyway.